app/models/card.rb

```diff
@@ -1,9 +1,11 @@
 class Card < ActiveRecord::Base
-validates :original_text, :translated_text, presence: true, uniqueness: { case_sensitive: false }
-validates :review_date, presence: true
-validate :must_not_be_equal
-scope :expired, -> { where('review_date <= ?', Date.today) }
-scope :random, -> { offset(rand(count))}
+  belongs_to :user
+  validates :original_text, :translated_text, presence: true, uniqueness: { case_sensitive: false }
+  validates :review_date, presence: true
+  validates :user_id, presence: true
+  validate :must_not_be_equal
+  scope :expired, -> { where('review_date <= ?', Date.today) }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.